commcare_connect/opportunity/visit_import.py

```diff
@@ -131,14 +131,11 @@ def _bulk_update_visit_status(opportunity: Opportunity, dataset: Dataset):
 
                 if visit.status != status:
                     visit.update_status(status)
+                    visit.reason = reason
                     if opportunity.managed and status in ["approved", "rejected"]:
                         visit.review_created_on = now()
                     changed = True
 
-                if status == VisitValidationStatus.rejected and reason and reason != visit.reason:
```

Comment:
I think this means changes to the reason after it has already been rejected will no longer be picked up. Is this intentional?

Reply:
This was a change to make the rejected reason field work as the justification. I reverted these changes in [7a3bfd8](https://github.com/dimagi/commcare-connect/pull/383/commits/7a3bfd8224a27efb62e59f98028645d5a44db573)